docs/checkers/index.md

```diff
@@ -2278,14 +2278,15 @@ if __name__ == "__main__":
     name = input()
 ```
 
-By default, [`input`], [`open`] and [`print`] are not allowed. However, you can choose which I/O functions specifically to disallow using the `forbidden-io-functions`
-option. This takes a list of function names that should not be used. For example,
-use the following configuration to forbid the use of [`print`] but allow [`input`] and [`open`]:
+By default, [`input`], [`open`] and [`print`] are not allowed. However, you can choose which I/O functions/methods
+specifically to disallow using the `forbidden-io-functions` option. This takes a list of function names and method
+[qualified names](https://peps.python.org/pep-3155/#proposal) that should not be used. For example, use the
+following configuration to forbid the use of [`print`] and `ClassName.method` but allow [`input`] and [`open`]:
```

Comment:
The text is good, but please rename to `MyClass.method` (I think this is slightly better at conveying the idea of a generic class)